content/webinars/building-self-service-clouds/index.md

```diff
@@ -31,7 +34,7 @@ external: false
 
 # The url slug for the webinar landing page. If this is an external
 # webinar, use the external URL as the value here.
-url_slug: "building-self-service-clouds-2020-04-22"
```

Comment:
Why go to the trouble of adding and then later removing these dates?

Reply:
Just a mistake in what I was doing. I didn't realize I was going to take off the dates from the URLs until I was figuring out how these pages were going to live.